.gitignore

```diff
@@ -0,0 +1,7 @@
+.vscode/
```

Comment:
Can you explain why this is here? I understand the need for a `.gitignore`, I'm specifically asking what is the relevance of this file for this PR and will this break any existing test? I would think this should be removed.

Reply:
I can rm it from this PR, not relevant for any of my changes